code/dropout.py

```diff
@@ -0,0 +1,447 @@
+from __future__ import print_function
+
+import numpy as np
+import os
+import sys
+import timeit
+import six.moves.cPickle as pickle
+import theano
+import theano.tensor as T
+import theano.tensor.shared_randomstreams
+import gzip
+from collections import OrderedDict
+from logistic_sgd import LogisticRegression, load_data
+
+
+
+class HiddenLayer(object):
```

Comment:
Yeah, I had wanted to do it in the previous commit, somehow slipped out of my mind. Will do